src/pcrlock/pcrlock.c

```diff
@@ -68,6 +68,7 @@ static bool arg_force = false;
 static BootEntryTokenType arg_entry_token_type = BOOT_ENTRY_TOKEN_AUTO;
 static char *arg_entry_token = NULL;
 static bool arg_varlink = false;
+static unsigned abbrev_hash = 7;
```

Comment:
maybe also add a comment `/* abbreviate to 7 chars, just like git */`

Reply:
So in git it's actually dynamic. It apparently calculates the minimum length that is still unique. Not sure I'd want to go that far here :-/